src/fixed-footer-layout.css.ts

```diff
@@ -29,6 +29,15 @@ export const fixedFooter = style({
     },
 });
 
+export const contentWithFixedFooter = style({
```

Comment:
alternative fix: make container be a new stacking context with `isolation: isolate`, this way zindex hierarchy inside container won't affect to the fixed footer.

It could be a good idea to apply `isolation: isolate` to cards too.

Reply:
Updated footer in order to use isolation instead of zIndex